runners/flink/job-server/flink_job_server.gradle

```diff
@@ -78,6 +79,7 @@ configurations.all {
 
 dependencies {
   compile project(flinkRunnerProject)
+  permitUnusedDeclared project(flinkRunnerProject) // BEAM-11761
```

Comment:
without this line, analyzeDependencies products an unusedDeclared error for all runners/flink/<version>/job-server/. We could have removed the compile line, but we can't until BEAM-11761 remains open.